assets/js/modules/analytics/components/common/TrackingExclusionSwitches.js

```diff
@@ -56,18 +83,30 @@ export default function TrackingExclusionSwitches() {
 				{ __( 'Exclude from Analytics', 'google-site-kit' ) }
 			</legend>
 
-			<div>
-				<Switch
-					label={ trackingExclusionLabels[ TRACKING_LOGGED_IN_USERS ] }
-					checked={ trackingDisabled.includes( TRACKING_LOGGED_IN_USERS ) }
-					onClick={ onChange }
-					hideLabel={ false }
-				/>
-			</div>
+			<Row>
+				<Cell>
```

Comment:
I think Row and Cell aren't really needed here. Let's wrap <Switche /> elements with just regular divs:

```html
<div>
    <Switch ...>
</div>
<div>
    <Switch ...>
</div>
```

Reply:
There is an AC about having them aligned right and left:
> The toggle for contentCreators should appear right of the existing toggle for loggedinUsers.

...Using `div` doesn't work, I guess as they are block level elements, so they'd need floating etc. to align them. 

| Using `div` | Using `Row` / `Cell` |
| --- | --- |
|  ![Screenshot 2021-05-04 at 12 00 40](https://user-images.githubusercontent.com/2470911/116994472-a8cb4180-acd0-11eb-949b-4a48dd5e83f0.png) | ![Screenshot 2021-05-04 at 12 01 51](https://user-images.githubusercontent.com/2470911/116994610-d617ef80-acd0-11eb-88bf-a0ed5fe9e401.png) |

Reply:
Ah, ok... I did miss this point. Then we need to wrap the row element with `<Grid>` and add appropriate size attributes for cell elements to make sure they are displayed next to each other on desktop and tablet screens, and stacked on mobile devices.